### ISSUE TITLE
Polkadot mainnet stops sync at block 2008592 for 'tokio-runtime-worker' panic

### ISSUE THREAD

**Issue by juno-yu-crypto** [paritytech/polkadot#1813]
```
Oct 14 01:30:44.021 ERROR Handler initialization process is too long with PeerId("12D3KooWKe1R7jLvnvEpoeX4dBZf2YYZukppmRSiStxNLfqqehPa")
Oct 14 01:30:45.029 ERROR Handler initialization process is too long with PeerId("12D3KooWF1KZeKQLS4RwawUxsGBfo9JqwstFA7axpEQQ28G83pSk")
Oct 14 01:30:45.085 ERROR Handler initialization process is too long with PeerId("12D3KooWHM7UcikhwjPqMmWoSHMxHzqjpfKVhPMtQm2sozWMR9jH")
Oct 14 01:30:45.731 ERROR Handler initialization process is too long with PeerId("12D3KooWE4A7AUtPYrRr75RB5pXi29PfBREdA1D8gmDD3H3yeXrn")
Oct 14 01:30:49.770 ERROR Handler initialization process is too long with PeerId("12D3KooWNseYZjN54jBwD9kd1zQeBhmX8gmZyeg1VWKqUN24to3d")
Oct 14 01:30:53.041 ERROR Handler initialization process is too long with PeerId("12D3KooWGsHSMtTbaSwg9y6v6RbVoQ2gBw9kDESta9uRt1gTKBmG")
Oct 14 01:30:53.296 ERROR Handler initialization process is too long with PeerId("12D3KooWJVsM5YGqBifL1voSNCvt5GRi8ozwnPMswk16tArCcGMs")
Oct 14 01:30:53.681 ERROR Handler initialization process is too long with PeerId("12D3KooWQiJj98qfCe2Zd6EPsGpKPct3vZDvFc8ZHmLa9xrz6qFS")
Oct 14 01:30:55.016  INFO ⚙️  Syncing  0.0 bps, target=#2013160 (13 peers), best: #2008592 (0x5e97…afd3), finalized #2008592 (0x5e97…afd3), ⬇ 18.4kiB/s ⬆ 4.1kiB/s

====================

Version: 0.8.24-9d6cd80-x86_64-linux-gnu

   0: sp_panic_handler::set::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/libstd/panicking.rs:530
   2: std::panicking::begin_panic
   3: frame_executive::Executive<System,Block,Context,UnsignedValidator,AllModules,COnRuntimeUpgrade>::execute_extrinsics_with_book_keeping
   4: tracing::span::Span::in_scope
   5: frame_executive::Executive<System,Block,Context,UnsignedValidator,AllModules,COnRuntimeUpgrade>::execute_block
   6: <polkadot_runtime::Runtime as sp_api::runtime_decl_for_Core::Core<sp_runtime::generic::block::Block<sp_runtime::generic::header::Header<u32,sp_runtime::traits::BlakeTwo256>,sp_runtime::generic::unchecked_extrinsic::UncheckedExtrinsic<<<sp_runtime::MultiSignature as sp_runtime::traits::Verify>::Signer as sp_runtime::traits::IdentifyAccount>::AccountId,polkadot_runtime::Call,sp_runtime::MultiSignature,(frame_system::extensions::check_spec_version::CheckSpecVersion<polkadot_runtime::Runtime>,frame_system::extensions::check_tx_version::CheckTxVersion<polkadot_runtime::Runtime>,frame_system::extensions::check_genesis::CheckGenesis<polkadot_runtime::Runtime>,frame_system::extensions::check_mortality::CheckMortality<polkadot_runtime::Runtime>,frame_system::extensions::check_nonce::CheckNonce<polkadot_runtime::Runtime>,frame_system::extensions::check_weight::CheckWeight<polkadot_runtime::Runtime>,pallet_transaction_payment::ChargeTransactionPayment<polkadot_runtime::Runtime>,polkadot_runtime_common::claims::PrevalidateAttests<polkadot_runtime::Runtime>)>>>>::execute_block
   7: sp_api::runtime_decl_for_Core::execute_block_native_call_generator::{{closure}}
   8: std::thread::local::LocalKey<T>::with
   9: sc_executor::native_executor::WasmExecutor::with_instance::{{closure}}
  10: sc_executor::wasm_runtime::RuntimeCache::with_instance
  11: <sc_executor::native_executor::NativeExecutor<D> as sp_core::traits::CodeExecutor>::call
  12: sp_state_machine::execution::StateMachine<B,H,N,Exec>::execute_aux
  13: sp_state_machine::execution::StateMachine<B,H,N,Exec>::execute_using_consensus_failure_handler
  14: <sc_service::client::call_executor::LocalCallExecutor<B,E> as sc_client_api::call_executor::CallExecutor<Block>>::contextual_call
  15: <sc_service::client::client::Client<B,E,Block,RA> as sp_api::CallApiAt<Block>>::call_api_at
  16: sp_api::runtime_decl_for_Core::execute_block_call_api_at
  17: sp_api::Core::execute_block_with_context
  18: <&sc_service::client::client::Client<B,E,Block,RA> as sp_consensus::block_import::BlockImport<Block>>::import_block
  19: <sc_finality_grandpa::import::GrandpaBlockImport<BE,Block,Client,SC> as sp_consensus::block_import::BlockImport<Block>>::import_block
  20: <sc_consensus_babe::BabeBlockImport<Block,Client,Inner> as sp_consensus::block_import::BlockImport<Block>>::import_block
  21: sp_consensus::import_queue::import_single_block_metered
  22: <futures_util::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
  23: <futures_util::future::future::Map<Fut,F> as core::future::future::Future>::poll
  24: <futures_util::future::future::flatten::Flatten<Fut,<Fut as core::future::future::Future>::Output> as core::future::future::Future>::poll
  25: <futures_util::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
  26: <sc_service::task_manager::prometheus_future::PrometheusFuture<T> as core::future::future::Future>::poll
  27: <futures_util::future::select::Select<A,B> as core::future::future::Future>::poll
  28: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  29: std::thread::local::LocalKey<T>::with
  30: futures_executor::local_pool::block_on
  31: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
  32: <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
  33: tokio::runtime::task::harness::Harness<T,S>::poll
  34: tokio::runtime::blocking::pool::Inner::run
  35: tokio::runtime::context::enter
  36: std::sys_common::backtrace::__rust_begin_short_backtrace
  37: core::ops::function::FnOnce::call_once{{vtable.shim}}
  38: <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/liballoc/boxed.rs:1081
      <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/liballoc/boxed.rs:1081
      std::sys::unix::thread::Thread::new::thread_start
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/libstd/sys/unix/thread.rs:87
  39: start_thread
  40: clone


Thread 'tokio-runtime-worker' panicked at 'Transaction would exhaust the block limits', /rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/libstd/macros.rs:13

This is a bug. Please report it at:

	https://github.com/paritytech/polkadot/issues/new


====================

Version: 0.8.24-9d6cd80-x86_64-linux-gnu

   0: sp_panic_handler::set::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/libstd/panicking.rs:530
   2: rust_begin_unwind
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/libstd/panicking.rs:437
   3: core::panicking::panic_fmt
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/libcore/panicking.rs:85
   4: core::option::expect_none_failed
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/libcore/option.rs:1269
   5: <&mut dyn sp_externalities::Externalities as sp_io::crypto::Crypto>::start_batch_verify_version_1
   6: std::thread::local::LocalKey<T>::with
   7: sp_io::crypto::start_batch_verify_version_1
   8: <<sp_io::crypto::HostFunctions as sp_wasm_interface::HostFunctions>::host_functions::ExtCryptoStartBatchVerifyVersion1 as sp_wasm_interface::Function>::execute
   9: scoped_tls::ScopedKey<T>::with
  10: sc_executor_wasmtime::imports::call_static
  11: wasmtime::func::Func::new::{{closure}}
  12: wasmtime::trampoline::func::stub_fn
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: RegisterSetjmp
  18: wasmtime_runtime::traphandlers::catch_traps
  19: wasmtime::func::Func::call
  20: scoped_tls::ScopedKey<T>::set
  21: <sc_executor_wasmtime::runtime::WasmtimeInstance as sc_executor_common::wasm_runtime::WasmInstance>::call
  22: std::panicking::try
  23: std::thread::local::LocalKey<T>::with
  24: sc_executor::native_executor::WasmExecutor::with_instance::{{closure}}
  25: sc_executor::wasm_runtime::RuntimeCache::with_instance
  26: <sc_executor::native_executor::NativeExecutor<D> as sp_core::traits::CodeExecutor>::call
  27: sp_state_machine::execution::StateMachine<B,H,N,Exec>::execute_aux
  28: sp_state_machine::execution::StateMachine<B,H,N,Exec>::execute_using_consensus_failure_handler
  29: <sc_service::client::call_executor::LocalCallExecutor<B,E> as sc_client_api::call_executor::CallExecutor<Block>>::contextual_call
  30: <sc_service::client::client::Client<B,E,Block,RA> as sp_api::CallApiAt<Block>>::call_api_at
  31: sp_api::runtime_decl_for_Core::execute_block_call_api_at
  32: sp_api::Core::execute_block_with_context
  33: <&sc_service::client::client::Client<B,E,Block,RA> as sp_consensus::block_import::BlockImport<Block>>::import_block
  34: <sc_finality_grandpa::import::GrandpaBlockImport<BE,Block,Client,SC> as sp_consensus::block_import::BlockImport<Block>>::import_block
  35: <sc_consensus_babe::BabeBlockImport<Block,Client,Inner> as sp_consensus::block_import::BlockImport<Block>>::import_block
  36: sp_consensus::import_queue::import_single_block_metered
  37: <futures_util::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
  38: <futures_util::future::future::Map<Fut,F> as core::future::future::Future>::poll
  39: <futures_util::future::future::flatten::Flatten<Fut,<Fut as core::future::future::Future>::Output> as core::future::future::Future>::poll
  40: <futures_util::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
  41: <sc_service::task_manager::prometheus_future::PrometheusFuture<T> as core::future::future::Future>::poll
  42: <futures_util::future::select::Select<A,B> as core::future::future::Future>::poll
  43: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  44: std::thread::local::LocalKey<T>::with
  45: futures_executor::local_pool::block_on
  46: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
  47: <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
  48: tokio::runtime::task::harness::Harness<T,S>::poll
  49: tokio::runtime::blocking::pool::Inner::run
  50: tokio::runtime::context::enter
  51: std::sys_common::backtrace::__rust_begin_short_backtrace
  52: core::ops::function::FnOnce::call_once{{vtable.shim}}
  53: <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/liballoc/boxed.rs:1081
      <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/liballoc/boxed.rs:1081
      std::sys::unix::thread::Thread::new::thread_start
             at rustc/04488afe34512aa4c33566eb16d8c912a3ae04f9/src/libstd/sys/unix/thread.rs:87
  54: start_thread
  55: clone


Thread 'tokio-runtime-worker' panicked at 'Failed to register required extension: `VerificationExt`: ExtensionAlreadyRegistered', /usr/local/cargo/git/checkouts/substrate-7e08433d4c370a21/5ea2399/primitives/io/src/lib.rs:528

This is a bug. Please report it at:

	https://github.com/paritytech/polkadot/issues/new

```

**Comment by juno-yu-crypto** [comment]
occurs in  master-0.8.24-9d6cd80-2f48441d image ,

seems resolved when upgrading to `parity/polkadot:v0.8.25@sha256:c99cd036075c0144b80b2ce1c34dee277415973749dd869f8720d64f01b0f6a6`

but does this make 0.8.25 upgrade mandatory?

**Comment by bkchr** [comment]
There was a bug that got fixed in 0.8.25.
